Gemfile

```diff
@@ -19,6 +19,7 @@ gem "redcarpet"
 gem "rinku", require: "rails_rinku"
 gem "sassc-rails"
 gem "select2-rails"
+gem "sprockets-rails"
```

Comment:
Could sprockets be added in a separate commit, so we have a commit message explaining why it was added as a dependency?

Reply:
Annoyingly I actually did have this in a separate commit and then squashed it cause I wasn't sure it seemed worth leaving separate, I can rewrite that if necessary though.